samples/csharp_dotnetcore/adaptive-dialog/01.multi-turn-prompt/Dialogs/RootDialog.lg

```diff
@@ -8,7 +8,13 @@
 - Thanks, ${CapitalizeName(user.userProfile.Name)}.
 
 # CapitalizeName(param)
-- ${concat(toUpper(substring(param, 0, 1)), substring(param, 1))}
+- IF : ${count(param) != 1}
+    - ${concat(CapitalizeFirstLetter(param), substring(param, 1))}
```

Comment:
>concat [](start = 8, length = 6)

FWIW, I have been making heavy use of ` rather than concat--helps readibility for me.

Reply:
addressed.